### PR TITLE
[docs] SAML

### PR DESCRIPTION
**Pull request by withshubh**

### CODE REVIEW

File: docs/docs/user-authentication/sso/saml.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  
+  <div style={{textAlign: 'center'}}>
+
+  <img className="screenshot-full" src="/img/sso/saml/url.png" alt="SSO :SAML"/>
+
+  </div>
+
+6. The **Login URL** obtained can be utilized for accessing the workspace. Please note that ToolJet supports SAML login at the workspace level and not at the instance level. Thus, users will be logged in specifically to the chosen workspace.
+
+7. Users can now login to your workspace using the **Login URL** obtained in the previous step. The login page will display the name of the SAML provider configured in the workspace settings.
```
Comment (thehacklady) [inline docs/docs/user-authentication/sso/saml.md:49]: can you recheck this statement? 
I believe it should be their workspace in place of 'your workspace'

Comment (withshubh) [reply docs/docs/user-authentication/sso/saml.md:55]: It is correct.

File: docs/docs/user-authentication/sso/saml.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  <div style={{textAlign: 'center'}}>
+
+  <img className="screenshot-full" src="/img/sso/saml/enable.png" alt="SSO :SAMP" />
+
+  </div>
+
+3. Enter the following configuration details:
+
+  - **SAML Provider Name**: Enter the name of your SAML provider. This name will be displayed on the login page.
+  - **Identity provider metadata**: Upload the data from the metadata file provided by your SAML provider. This file contains the SAML configuration details.
```
Comment (thehacklady) [inline docs/docs/user-authentication/sso/saml.md:31]: @withshubh The part where user has to add IdP data needs bit more detail on how will the user find it and add it. Sharing the text for now, but let's try to keep this in mind in general on how we are adding information.

Updated text - 

To export the metadata as an XML file from your identity provider, initiate the download by clicking a button on your IdP dashboard. After downloading, copy the data from the XML file.

Paste the copied metadata into ToolJet's SAML SSO configuration settings. Please ensure that the metadata is pasted in the correct format, as it contains essential configuration details from the identity provider necessary for authentication.

Additionally, you can often find this data by navigating to https://<provider-domain>/federationmetadata/2007-06/federationmetadata.xml

File: docs/docs/user-authentication/sso/saml.md (hunk 1 above)
Comment (thehacklady) [reply docs/docs/user-authentication/sso/saml.md:55]: 1. lets have only 2 points in login section
2. point 2 needs to be rephrased to this - The Login URL obtained can be used to access the workspace. Please note that ToolJet supports SAML login at the workspace level, ensuring users are logged in specifically to the selected workspace.

As a result, users can now log in to your workspace using the provided Login URL. The login page will prominently feature the name of the SAML provider configured in your workspace settings.

**Review by gsmithun4 — APPROVED** [review]

**Pull request merged**